modules/i18n/input-method/default.nix

```
@@ -0,0 +1,67 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+let
+  cfg = config.i18n.inputMethod;
+
+  gtk2_cache = pkgs.runCommandLocal "gtk2-immodule.cache" {
+    buildInputs = [ pkgs.gtk2 cfg.package ];
+  } ''
+    mkdir -p $out/etc/gtk-2.0/
+    GTK_PATH=${cfg.package}/lib/gtk-2.0/ gtk-query-immodules-2.0 > $out/etc/gtk-2.0/immodules.cache
+  '';
+
+  gtk3_cache = pkgs.runCommandLocal "gtk3-immodule.cache" {
+    buildInputs = [ pkgs.gtk3 cfg.package ];
+  } ''
+    mkdir -p $out/etc/gtk-3.0/
+    GTK_PATH=${cfg.package}/lib/gtk-3.0/ gtk-query-immodules-3.0 > $out/etc/gtk-3.0/immodules.cache
+  '';
+
+in {
+  imports =
+    [ ./fcitx.nix ./fcitx5.nix ./hime.nix ./kime.nix ./nabi.nix ./uim.nix ];
+
+  options.i18n = {
+    inputMethod = {
+      enabled = mkOption {
+        type = types.nullOr
+          (types.enum [ "fcitx" "fcitx5" "nabi" "uim" "hime" "kime" ]);
+        default = null;
+        example = "fcitx";
+        description = ''
+          Select the enabled input method. Input methods is a software to input symbols that are not available on standard input devices.
+
```

Unfortunately need
```suggestion
          </para><para>
```
for paragraphs.

---

Got it.

---

I think this is still a problem.

---

That's not needed for this module is it? It looks good.

---

I don't know, it seems more readable to me to have separate paragraphs. But I don't particularly mind, but if you intentionally don't want paragraphs here the empty lines should be removed. Something like
```
          Select the enabled input method. Input methods is a software to input
          symbols that are not available on standard input devices. Input
          methods are specially used to input Chinese, Japanese and Korean
          characters. Currently the following input methods are available in
          Home Manager:
```
Should be OK.